content/articles/reverse-dns.markdown

```diff
@@ -1,12 +1,12 @@
 ---
-title: Do you support Reverse DNS (i.e. PTR records)?
-excerpt: 
+title: Reverse DNS zones
+excerpt: We support reverse DNS zones and we can enable the feature on request.
 categories:
 - DNS
 ---
 
-# Do you support Reverse DNS (i.e. PTR records)?
+# Reverse DNS zones
 
-DNSimple does support reverse DNS entries however we do not enable this feature by default on customer accounts. We'll be happy to enable it for you, however first please ensure that you will actually be able to delegate the portion of the subnet that you would like us to respond for reverse DNS queries.
+DNSimple does support reverse DNS entries (i.e. PTR records) however we do not enable this feature by default on customer accounts. We'll be happy to enable it for you, however first please ensure that you will actually be able to delegate the portion of the subnet that you would like us to respond for reverse DNS queries.
```

Comment:
Yes, it's a good point. I believe we added it in the past somewhere, I thought it was here. Or was it a different feature?

We also have a specific feature banner (see https://support.dnsimple.com/articles/regional-records/), I think we should use it for consistency.